libraries/RC_Channel/RC_Channel.h

```diff
@@ -170,7 +170,9 @@ class RC_Channel {
         ALTHOLD   =           70, // althold mode
         FLOWHOLD  =           71, // flowhold mode
         CIRCLE    =           72, // circle mode
-        DRIFT     =           73  // drift mode
+        DRIFT     =           73, // drift mode
+        KILL_IMU1 =          100,
```

Comment:
We should add these to the parameter description in lines 87 -89 of RC_Channel.cpp

Reply:
fixed